### PR TITLE
[SD-3031] Updates of images from Reuters causes media entries to beco…

### PR DESCRIPTION
**Pull request by marwoodandrew**
…me orphaned

It seems that the Reuters feed repeatedly ingests the same images, in this case the image would be downloaded again and the ingest item replaced in mongo. The old Grid FS entry would be orphaned.

I did some testing downloading the image with the update, calculating the md5 and downloading again if different. It never was so I simplified the code just to check if we already had the image.

### CODE REVIEW

File: superdesk/io/commands/update_ingest.py [hunk 1]
```diff
@@ -382,11 +382,32 @@ def ingest_item(item, provider, rule_set=None, routing_scheme=None):
...
     inserted = []
     try:
+        # If there is an existing set of renditions we keep those
+        if old_item:
+            media = old_item.get('renditions', {}).get('original', {}).get('media', {})
+            if media:
+                item['renditions'] = old_item['renditions']
+                item['mimetype'] = old_item.get('mimetype')
+                item['filemeta'] = old_item.get('filemeta')
+                logger.info("Reuters image not updated for GUID:{}".format(item[GUID_FIELD]))
```
Comment (amagdas) [inline superdesk/io/commands/update_ingest.py]: Is this specific only to Reuters or the message can be more generic?

**Comment by amagdas** [comment]
:+1:

File: superdesk/io/commands/update_ingest.py (hunk 1 above)
Comment (marwoodandrew) [inline superdesk/io/commands/update_ingest.py]: Only Reuters at the moment, but probably should be generic, it is core :-)

**Comment by mdhaman** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**